resolwe_bio/descriptors/general_clinical.yml

```diff
@@ -0,0 +1,1135 @@
+- slug: general-clinical
+  name: General clinical annotation
+  description: General clinical metadata schema.
+  version: 1.0.0
+  schema:
+    - name: general
+      label: General
+      group:
+        - name: species
+          label: Species
+          type: basic:string
+          default: Homo sapiens
+          required: false
+        - name: annotator
+          label: Annotator
+          type: basic:string
+          required: false
+    - name: subject_information
+      label: Subject information
+      group:
+        - name: sample
+          label: Sample
+          type: basic:string
+          required: false
+        - name: subject_id
+          label: Subject ID
+          type: basic:string
+          required: false
+        - name: batch
+          label: Batch
+          type: basic:integer
+          required: false
+        - name: group
+          label: Group
+          type: basic:integer
+          required: false
+        - name: gender
+          label: Gender
+          type: basic:string
+          required: false
+          choices:
+            - label: M
+              value: m
+            - label: F
+              value: f
+        - name: race
+          label: Race
+          type: basic:string
+          required: false
+          choices:
+            - label: White
+              value: white
+            - label: Non-White
+              value: non-white
+            - label: Asian
+              value: asian
+            - label: Non-Asian
+              value: non-asian
+            - label: Black
+              value: black
+            - label: Non-Black
+              value: non-black
+        - name: ethnicity
+          label: Ethnicity
+          type: basic:string
+          required: false
+          choices:
+            - label: African Americans
+              value: african americans
+            - label: Non-African Americans
+              value: non-african americans
+            - label: Hispanic
+              value: hispanic
+            - label: Non-Hispanic
+              value: non-hispanic
+            - label: American Indians and Alaska Natives
+              value: american indians and alaska natives
+            - label: Asians and Pacific Islanders
+              value: asians and pacific islanders
+            - label: Non-Asians and Pacific Islanders
+              value: non-asians and pacific islanders
+        - name: age_in_years
+          label: Age in years
+          type: basic:decimal
+          required: false
+        - name: height
+          label: Height
+          type: basic:decimal
+          required: false
+        - name: weight
+          label: Weight
+          type: basic:decimal
+          required: false
+        - name: archival
```

Comment:
Perhaps this belongs more to `Sample preparation and processing` than to `Subject information`. Instead of the options `yes` and `no` it may make more sense to have the options `fresh` and `archival` and adjust the name of the field.